packages/opentelemetry-api/README.md

```diff
@@ -9,12 +9,112 @@ This package provides everything needed to interact with the OpenTelemetry API,
 
 ## Basic Use
 
-### API Entry Point
+To get started tracing you need to install the SDK and plugins, create a TracerProvider, and register it with the API.
+
+### Install Dependencies
+
+```sh
+$ npm install \
+    @opentelemetry/core \
+    @opentelemetry/node \
+    @opentelemetry/tracing \
+    @opentelemetry/exporter-jaeger \
+    @opentelemetry/exporter-prometheus \ # add exporters as needed
+    @opentelemetry/plugin-http # add plugins as needed
+```
+
+### Initialize the SDK
+
+Before any other module in your application is loaded, you must initialize the global tracer and meter providers. If you fail to initialize a provider, no-op implementations will be provided to any library which acquires them from the API.
+
+To collect traces and metrics, you will have to tell the SDK where to export telemetry data to. This example uses Jaeger and Prometheus, but exporters exist for other tracing backends. If you're not sure if there is an exporter for your tracing backend, contact your tracing provider.
```

Comment:
Add link to highlight other tracing backends : https://github.com/open-telemetry/opentelemetry-js#trace-exporters